Valve.SteamLink version 1.3.15

SSLbits opened the pull request:
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.9 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.9.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/214232)

SSLbits committed:
Valve.SteamLink version 1.3.15

stephengillie commented:
Hi @SSLbits,

This manifest uses the same values for DisplayVersion and PackageVersion. This is not recommended, and the DisplayVersion should be removed.

(Automated response - build 949.)
<!--
[Policy] Needs-Author-Feedback
-->

SSLbits committed:
Update Valve.SteamLink.installer.yaml

Remove DisplayVersion field as requested by validation

SSLbits commented:
sorry about that @stephengillie
that manifest was created by wingetcreate so it automatically added those values. I've removed it and updated the PR.

stephengillie commented:
> that manifest was created by wingetcreate so it automatically added those values. I've removed it and updated the PR.

I've [added an Issue](https://github.com/microsoft/winget-create/issues/582) to address this.

mdanish-kh commented:
@SSLbits Can you let me know how you created the manifest using wingetcreate? :) That'll help identify the issue

SSLbits commented:
![image](https://github.com/user-attachments/assets/f0d66133-cf3a-4736-a534-d7e20f10ce5b)
@mdanish-kh

mdanish-kh commented:
Thanks @SSLbits - that helps a lot! This should only be happening right now with MSIs and only in the "interactive" update flow
